documents/invoice/model.go

```diff
@@ -624,6 +624,15 @@ func (i *Invoice) unpackFromUpdatePayload(payload documents.UpdatePayload) (*Inv
 	}, nil
 }
 
+// Patch merges payload data into model
+func (i *Invoice) Patch(payload documents.UpdatePayload) (documents.Model, error) {
+	inv, err := i.unpackFromUpdatePayload(payload)
```

Comment:
I see a problem here. `unpackFromUpdatePayload` internally calls `PreparesNewVersion` making it skip the current version without anchoring it.

I think we need to split the unpackFromUpdatePayload to patching and preparing new version. WDYT ?

Reply:
you are right, didn't catch that, I will separate those now.